index.bs

```diff
@@ -4646,6 +4646,13 @@ Registry Definitions</h4>
 			The [=custodian=] may be the [=Working Group=], the [=Team=], or a delegated entity.
 			The [=custodian=] for all [=registry tables=] in a single [=registry=]
 			<em class=rfc2119>should</em> generally be the same entity.
+
+			If a [=custodian=] of a [=registry=] becomes defunct
+			(e.g., the relevant group is disbanded),
```

Comment:
(or the custodian is unresponsive to repeated attempts to make contact)

Reply:
```suggestion
			(e.g., the relevant group is disbanded, or 
			the custodian is unresponsive to repeated attempts to make contact),
```